DQMOffline/Trigger/plugins/BPHMonitor.h

```diff
@@ -80,26 +96,94 @@ class BPHMonitor : public DQMEDAnalyzer
 
   edm::EDGetTokenT<reco::MuonCollection>        muoToken_;
   edm::EDGetTokenT<reco::BeamSpot>        bsToken_;
+  edm::EDGetTokenT<reco::TrackCollection>       trToken_;
+  edm::EDGetTokenT<reco::PhotonCollection>       phToken_;
 
   MEbinning           phi_binning_;
   MEbinning           pt_binning_;
   MEbinning           eta_binning_;
   MEbinning           d0_binning_;
   MEbinning           z0_binning_;
-  MEbinning           ls_binning_;
+  MEbinning           dR_binning_;
+  MEbinning           mass_binning_;
+  MEbinning           dca_binning_;
+  MEbinning           ds_binning_;
+  MEbinning           cos_binning_;
+  MEbinning           prob_binning_;
 
   METME muPhi_;
   METME muEta_;
   METME muPt_;
   METME mud0_;
   METME muz0_;
 
+  METME mu1Phi_;
+  METME mu1Eta_;
+  METME mu1Pt_;
+  METME mu1d0_;
+  METME mu1z0_;
+  METME mu2Phi_;
+  METME mu2Eta_;
+  METME mu2Pt_;
+  METME mu2d0_;
+  METME mu2z0_;
+  METME mu3Phi_;
+  METME mu3Eta_;
+  METME mu3Pt_;
+  METME mu3d0_;
+  METME mu3z0_;
+
+
+///
+METME  phPhi_   ;
+METME  phEta_   ;
+METME  phPt_   ;
+METME  DiMuPhi_   ;
+METME  DiMuEta_   ;
+METME  DiMuPt_   ;
+METME  DiMuPVcos_   ;
+METME  DiMuProb_   ;
+METME  DiMuDS_   ;
+METME  DiMuDCA_   ;
+METME  DiMuMass_   ;
+METME  DiMudR_   ;
+
+
+//
 
   GenericTriggerEventFlag* num_genTriggerEventFlag_;
   GenericTriggerEventFlag* den_genTriggerEventFlag_;
   StringCutObjectSelector<reco::Muon,true>        muoSelection_;
   StringCutObjectSelector<reco::Muon,true>        muoSelection_ref;
+  StringCutObjectSelector<reco::Muon,true>        muoSelection_tag;
+  StringCutObjectSelector<reco::Muon,true>        muoSelection_probe;
   int nmuons_;
+  int tnp_;
+  int L3_;
+  int trOrMu_;
+  int Jpsi_;
+  int Upsilon_;
+  int nofset_;
```

Comment:
Hi @anigamova - thanks for the cleaned up PR - its easier to make comments on now.

Could nofset_ be an enum instead ? This would let its values describe what they actually mean.